### PR TITLE
Touchmaster Delta Siren

### PR DESCRIPTION
**Pull request by Officer-Pringle**
Added a new Touchmaster Delta Siren,
This incudes a slightly louder .WAV and also a "jingle" Type of Phaser siren

- Horn - Schmal - Mastercom B
- Manual - Schmal - Touchmaster Delta

### CODE REVIEW

**Review by JoshPiper — DISMISSED** [review]
The yelp siren is currently clipping. That needs to be resolved before I can approve this.

**Comment by SuperMighty1** [comment]
Is it possible to merge this with the current TM Delta siren to include the phaser before the scan tone as that seems to be the only significant difference between them?

**Comment by Officer-Pringle** [comment]
Would it be fine to do what SM Said and just add the phaser to the photon version of the Touchmaster Delta

File: lua/autorun/photon/library/emv_sirens.lua [hunk 1]
```diff
@@ -36,6 +36,7 @@ local sirenTable = {
 		Set = {
 			{Name = "WAIL", Sound = "emv/sirens/federal sig tm/emv_wail.wav", Icon="wail"},
 			{Name = "YELP", Sound = "emv/sirens/federal sig tm/emv_yelp.wav", Icon="yelp"},
+			{Name = "PHSR", Sound = "sound/emv/sirens/p_fs_touchmaster_delta/pemv_tmjingle.wav", Icon = "phaser"} -- Edit by Pringle
```
Comment (JoshPiper) [inline lua/autorun/photon/library/emv_sirens.lua]: This is missing a comma.

**Pull request closed without merging**